.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +0,0 @@
-<!-- PLEASE ENSURE YOU REVIEW THE CONTENT OF EACH PR CAREFULLY, INCLUDING SUBSEQUENT COMMENTS BY YOURSELF OR OTHERS. -->
```

Comment:
Not sure what happened here, but these files should not be deleted.

Reply:
Seconded